resources/gift-card-adjustment.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const assign = require('lodash/assign');
+const omit = require('lodash/omit');
+
+const baseChild = require('../mixins/base-child');
+
+/**
+ * Creates a GiftCardAdjustment instance.
+ *
+ * @param {Shopify} shopify Reference to the Shopify instance
+ * @constructor
+ * @public
+ */
+function GiftCardAdjustment(shopify) {
+  this.shopify = shopify;
+
+  this.parentName = 'gift_cards';
+  this.name = 'adjustments';
+  this.key = 'adjustment';
+}
+
+assign(GiftCardAdjustment.prototype, omit(baseChild, ['count', 'delete', 'update']));
+
+/**
+ * Gets a list of adjustments for a given gift card.
+ *
+ * @param {Number} giftCardId Gift Card ID
+ * @return {Promise} Promise that resolves with the result
+ * @public
+ */
+GiftCardAdjustment.prototype.list = function list(customerId) {
```

Comment:
Any reason for overriding this? 

https://github.com/MONEI/Shopify-api-node/blob/2a252502d12308d68080f0c4714744ce38900a71/mixins/base-child.js#L76-L79

Should work no?